terraform/backend_uk.tf

```diff
@@ -0,0 +1,147 @@
+module "backend_uk" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.2.0"
+
+  service_name  = var.service_name
+  service_count = var.service_count
+  environment   = var.environment
+  region        = var.region
+
+  cluster_name     = "trade-tariff-cluster-${var.environment}"
+  subnet_ids       = data.aws_subnets.private.ids
+  security_groups  = [data.aws_security_group.this.arn]
+  target_group_arn = data.aws_lb_target_group.this.arn
+
+  min_capacity = var.min_capacity
+  max_capacity = var.max_capacity
+  docker_image = data.aws_ssm_parameter.ecr_url.value
+  docker_tag   = var.docker_tag
+  skip_destroy = true
+
+  service_environment_config = [
+    {
+      name  = "ALLOW_MISSING_MIGRATION_FILES"
+      value = "true"
+    },
+    {
+      name  = "AWS_BUCKET_NAME"
+      value = "trade-tariff-persistence-${var.environment}"
+    },
+    {
+      name  = "BETA_SEARCH_MAX_HITS"
+      value = "1000"
+    },
+    {
+      name  = "DB_POOL"
+      value = "20"
+    },
+    {
+      name  = "FRONTEND_HOST"
+      value = "https://${local.environment_key}.${local.base_url}/"
+      # TODO: scope this due to www
+    },
+    {
+      name  = "MALLOC_ARENA_MAX"
+      value = "2"
+    },
+    {
+      name  = "MAX_THREADS"
+      value = "6"
+    },
+    {
+      name  = "NEW_RELIC_AGENT_ENABLED"
+      value = "false"
+    },
+    {
+      name  = "NEW_RELIC_APP_NAME"
+      value = "tariff-uk-backend-${var.environment}"
+    },
+    {
+      name  = "NEW_RELIC_DISTRIBUTED_TRACING"
+      value = "true"
+    },
+    {
+      name  = "PAAS_S3_SERVICE_NAME"
+      value = "tariff-pdf-${local.environment_key}"
```

Comment:
Looks like an S3 bucket so probably want to create this in the core infrastructure and then fetch the name with a data source